cmd/jiralert/content.go

```diff
@@ -99,6 +99,12 @@ func pageTemplate(name string) *template.Template {
 // HomeHandlerFunc is the HTTP handler for the home page (`/`).
 func HomeHandlerFunc() func(http.ResponseWriter, *http.Request) {
 	return func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != "GET" {
+			w.WriteHeader(http.StatusBadRequest)
+			w.Write([]byte("only GET allowed")) //nolint:errcheck
```

Comment:
```suggestion
			_,  _ = w.Write([]byte("only GET allowed")) 
```

This will do the work for explicit error non-handling (:

Reply:
yes, but vscode dev-tooling starts complaining then:

![image](https://user-images.githubusercontent.com/5539202/110112755-6f4e8680-7db2-11eb-80c5-3f646cecc547.png)

Reply:
oops, I screenshotted another snippet. but with the same symptoms :shrug:

Reply:
Because this is wrong and won't even compile (: 

I mean this:

`_,  _ = w.Write([]byte("only GET allowed"))`

(No semicolon)

Reply:
:facepalm: thanks!